storage-model/src/main/java/no/unit/nva/publication/storage/model/DatabaseConstants.java

```diff
@@ -38,8 +49,25 @@
         ENTRY_IDENTIFIER);
     public static final String BY_TYPE_CUSTOMER_STATUS_SK_FORMAT =
         String.join(KEY_FIELDS_DELIMITER, RECORD_TYPE, ENTRY_IDENTIFIER);
-    
+
     private DatabaseConstants() {
-    
+
+    }
+
+    private static String readTableNameFromEnvironment() {
+        try {
+            return environment.readEnv(RESOURCES_TABLE_NAME_ENV_VARIABLE);
```

Comment:
What about: environment.readEnvOpt(RESOURCES_TABLE_NAME_ENV_VARIABLE).orElse(defaultValue());

Reply:
My intention was to log the lack of the env variable but maybe it is not necessary.

Reply:
I think it will log regardless, but might want to test first